pkg/client/unversioned/request.go

```diff
@@ -718,12 +745,8 @@ func (r *Request) Do() Result {
 	return result
 }
 
-// DoRaw executes the request but does not process the response body.
+// DoRaw executes the request (without processing the response body).
```

Comment:
Nit: Not really an improvement over the previous comment, just a change. Maybe just revert it? (same for line 730 above)